### ISSUE TITLE
The label element should be visible

### ISSUE THREAD

**Issue by kuzmenkn** [CDH-Studio/UpSkill#1716]
Some label elements are not visible. Clicking on the label doesn't activate the dropdown. 

Expected: Clicking on the label will activate the dropdown
To enlarge the clickable area the label element must be visible since it provides assistance to all users who need help understanding the purpose of the field. This can be helpful for users with impaired motor control.

WCAG 2.1
Success Criterion 3.3.2,

See images

![image](https://user-images.githubusercontent.com/90463917/140769724-e2d644ec-66d7-4d12-a647-8f05456a0b5f.png)
![image](https://user-images.githubusercontent.com/90463917/140769872-5f5a4e27-e443-4935-96bb-3c342cb6c754.png)
![image](https://user-images.githubusercontent.com/90463917/140770056-5b25dd77-acd2-4ffd-bb94-77efa48e514f.png)
